qiskit/compiler/transpiler.py

```diff
@@ -210,10 +210,12 @@ def transpile(
         optimization_level: How much optimization to perform on the circuits.
             Higher levels generate more optimized circuits,
             at the expense of longer transpilation time.
-            * 0: no optimization
-            * 1: light optimization
-            * 2: heavy optimization
-            * 3: even heavier optimization
+
+                * 0: no optimization
+                * 1: light optimization
+                * 2: heavy optimization
+                * 3: even heavier optimization
+
```

Comment:
The indentation isn't needed for bullet lists, they just need a blank line before the first and after the last bullet.  This one was missing a blank before, which is probably why it wasn't rendering.

Reply:
Indeed, fixed in 915c7e96b59b0721344d0c8f7bee61df1211d099